libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts

```diff
@@ -9,9 +9,14 @@ import { ElewaBrandsOpportunitiesComponent } from './components/elewa-brands-opp
 import { ElewaBrandsPageComponent } from './pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component';
 
 import { BrandsRoutingModule } from './brands.routing';
+import { ElewaBrandsItalantaComponent } from './components/elewa-brands-italanta/elewa-brands-italanta.component';
```

Comment:
order of imports

router should be imported last and pages are imported after components

Reply:
@IanOdhiambo9 Kindly review, did correction on order of imports